site/src/pages/ManagementSettingsPage/GroupsPage/GroupsPage.tsx

```diff
@@ -3,53 +3,69 @@ import Button from "@mui/material/Button";
 import { type FC, useEffect } from "react";
 import { Helmet } from "react-helmet-async";
 import { useQuery } from "react-query";
-import {
-  Navigate,
-  Link as RouterLink,
-  useLocation,
-  useParams,
-} from "react-router-dom";
+import { Navigate, Link as RouterLink, useParams } from "react-router-dom";
 import { getErrorMessage } from "api/errors";
 import { groups } from "api/queries/groups";
+import { organizationPermissions } from "api/queries/organizations";
 import type { Organization } from "api/typesGenerated";
+import { EmptyState } from "components/EmptyState/EmptyState";
 import { displayError } from "components/GlobalSnackbar/utils";
+import { Loader } from "components/Loader/Loader";
 import { PageHeader, PageHeaderTitle } from "components/PageHeader/PageHeader";
-import { useAuthenticated } from "contexts/auth/RequireAuth";
-import { useDashboard } from "modules/dashboard/useDashboard";
 import { useFeatureVisibility } from "modules/dashboard/useFeatureVisibility";
 import { pageTitle } from "utils/page";
 import { useOrganizationSettings } from "../ManagementSettingsLayout";
 import GroupsPageView from "./GroupsPageView";
 
 export const GroupsPage: FC = () => {
-  const { permissions } = useAuthenticated();
-  const { createGroup: canCreateGroup } = permissions;
-  const {
-    multiple_organizations: organizationsEnabled,
-    template_rbac: isTemplateRBACEnabled,
-  } = useFeatureVisibility();
-  const { experiments } = useDashboard();
-  const location = useLocation();
-  const { organization = "default" } = useParams() as { organization: string };
-  const groupsQuery = useQuery(groups(organization));
+  const feats = useFeatureVisibility();
+  const { organization: organizationName } = useParams() as {
+    organization: string;
+  };
+  const groupsQuery = useQuery(
+    organizationName ? groups(organizationName) : { enabled: false },
```

Comment:
TIL! I was doing:
```tsx
const groupsQuery = useQuery({
    ...groups(organizationName ?? ""),
    { enabled: !!organizationName }
})
```
But your way is better.

Reply:
I think I stole this from Kayla actually :laughing: